frontend/src/metabase/nav/components/ProfileLink.jsx

```diff
@@ -21,32 +22,27 @@ const mapStateToProps = state => ({
   adminItems: getAdminPaths(state),
 });
 
-@connect(mapStateToProps)
-export default class ProfileLink extends Component {
-  state = {
-    dropdownOpen: false,
-  };
+export default connect(mapStateToProps)(ProfileLink);
```

Comment:
I deliberately put the `export default` at the top not just because it's where it stayed previously, but because I think this should be the first thing we should look for in any given component. Every time I visit a file and don't see any `export default` usually they're at the bottom, so I'll need to scroll to the bottom to see what's being exported.

Feel free to discuss about this.